docker-compose.yml

```diff
@@ -42,8 +42,6 @@ services:
 
     customers-service-reactive:
         build: customers-service-reactive
-        ports:
-#            - "8090:8080"
```

Comment:
Why did you do that anyways?

Reply:
👍

Reply:
because @Nmazza1  was complaining about the ports so i wanted to keep it commented out so it would be easier for him. Clearly it was a mistake to try and make his life easier....